docs/Silver Blaze.html

```diff
@@ -14,2546 +15,1161 @@ <h1>
 	</h1>
 	<p>
 		I am afraid, Watson, that I shall have to go,” said Holmes, as we
-I am afraid, Watson, that I shall have to go,” said Holmes, as we
 		sat down together to our breakfast one morning.
-sat down together to our breakfast one morning.
 	</p>
-	<p>		
-
-		“Go! Where to?”
-“Go! Where to?”
+	<p>		“Go! Where to?”
```

Comment:
The text should be on the next line. Could you modify the code so that after running the tool, this file is unchanged?